lib/oscap.py

```diff
@@ -37,7 +37,10 @@ def make_rule():
             return types.SimpleNamespace(fixes=self.FixType(0))
         self.rules = collections.defaultdict(make_rule)
         if xml_file:
+            self.path = Path(xml_file)
             self.parse_xml(xml_file)
+        else:
+            self.path = None
```

Comment:
This is probably unnecessary, `util.get_datastream()` does the same thing and is more widely used.

I think the mental model around this should be
* `util.get_datastream()` --->  `oscap.global_ds()`

rather than `oscap.global_ds()` being the one-stop-shop for all things DS.